utils/execJob.js

```diff
@@ -1,8 +1,10 @@
 import sendTelegram from './sendTelegram.js'
 import templateBody from './templateBody.js'
+import delay from './delay.js'
 
 const execJob = async (job, done, payload) => {
   try {
+    await delay();
```

Comment:
kenapa tidak menggunakan delay dari bull ?

Reply:
delay from bull, only delays the execution time, but it takes time interval between processes :smile: 
the issue is giving delay time when sending telegrams